.github/dependabot.yml

```diff
@@ -1,15 +1,16 @@
 # https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
-
+---
```

Comment:
Making `yamllint(1)` happy.

Reply:
Or just delete the comment since the link is dead anyway!

Reply:
Pre-existing issue